src/exception/exception-service.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022 The Ontario Institute for Cancer Research. All rights reserved
+ * Copyright (c) 2023 The Ontario Institute for Cancer Research. All rights reserved
```

Comment:
choosing to update the year only in this file? 😉

this must be a clue... looking for an eastern egg now